README.md

```diff
@@ -20,31 +20,31 @@ A Github Action for uploading files to a webdav server
-| `webdav_upload_path`      | The WebDAV path where you want to upload, Some server not support root path                                                                                                        | -       |
-| [`files`](#files)         | Newline-delimited list of path globs for asset files to upload <br> :feet: You can learn more about multi-line yaml syntax [here](https://yaml-multiline.info/)                    | -       |
-| `fail_on_unmatched_files` | Fail the action when exist unmatched file pattern                                                                                                                                  | false   |
-| `keep_structure`          | Keep the directory structure of the files<br/> **Only support single search path**, [rule](https://github.com/actions/toolkit/blob/main/packages/glob/src/internal-globber.ts#L27) | false   |
-
-> :no_bicycles: Now we just support the basic authentication
+| Input                     | Description                                                                                                                                                                           | Default |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `webdav_address`          | WebDAV address                                                                                                                                                                        | -       |
+| `webdav_username`         | WebDAV username                                                                                                                                                                       | -       |
+| `webdav_password`         | WebDAV password                                                                                                                                                                       | -       |
+| `webdav_upload_path`      | The WebDAV path where you want to upload. Some servers may not support the root path                                                                                                  | -       |
+| [`files`](#files)         | Newline-delimited list of path globs for asset files to upload <br> :feet: You can learn more about multi-line YAML syntax [here](https://yaml-multiline.info/)                       | -       |
+| `fail_on_unmatched_files` | Fail the action if there exists an unmatched file pattern                                                                                                                             | false   |
+| `keep_structure`          | Keep the directory structure of the files<br/> **Only supports a single search path**, [rule](https://github.com/actions/toolkit/blob/main/packages/glob/src/internal-globber.ts#L27) | false   |
```

Comment:
I didn't quite understand this line. If you can explain further (in English or Japanese), I'll gladly improve it further.

Reply:
Initially, I want to upload some files under the specified directory. but in #51 somebody wants to keep struct like: 

> Local: `dist` =>  WebDAV: `public` (for deployment purposes I think)

I don't want to change a lot, so if set `keep_structure` the `files` only support one path

the code:

https://github.com/bxb100/action-upload-webdav/blob/9e4f49f597ccd11be3791374616bd67849634830/src/main.ts#L32-L37

https://github.com/bxb100/action-upload-webdav/blob/9e4f49f597ccd11be3791374616bd67849634830/src/main.ts#L54-L62